runtimes/server-interface/lsp.ts

```diff
@@ -94,6 +99,11 @@ export type Lsp = {
         getConfiguration: (section: string) => Promise<any>
         onDidChangeWorkspaceFolders: (handler: NotificationHandler<DidChangeWorkspaceFoldersParams>) => void
     }
+    window: {
+        showMessage: (params: ShowMessageParams) => Promise<void>
```

Comment:
You seem to be using only `showMessage` for now in the related PR. Do you plan to use the rest in the following PRs?

Reply:
I didn't plan to use them, but I don't see a reason why not to add them now, since I'm touching this feature. 
If there is a need in other 2 tomorrow, they will be readily available.